Add real life example to comms package readme (#67)
leancodepl/comms: packages/comms/README.md

}
```

### Real life example

Most common bloc that each app has is `AuthBloc` or `AuthCubit`. It's important
for all other blocs to know and react to whether user is authenticated or not.

> All you need to do is add a `StateSender` mixin which allows every other bloc
> to listen for `AuthState` changes.

Reviewer: ```suggestion
With `comms`, all you need to do is add a `StateSender` mixin to the auth bloc, which allows every other bloc to listen for `AuthState` changes.
```

```dart
class AuthCubit extends Cubit<AuthState> with StateSender {...}
```

> For example `UserProfileCubit` will fetch user's profile on sign in and
> should clear that data on sign out.

Reviewer: ```suggestion
For example `UserProfileCubit` will fetches user's profile on sign in and
clears that data on sign out.
```

Reviewer 2: ```suggestion
For example, `UserProfileCubit` fetches the user's profile on sign-in and
clears that data on sign-out.
```

```dart
class UserProfileCubit extends ListenerCubit<UserProfileState, AuthState> {
  UserProfileCubit({
    required UserRepository repository,
  }) : _repository = repository,
       super(const UserProfileState.initial());
  
  final UserRepository _repository;

  @override
  void onInitialMessage(AuthState message) => onMessage(message);

  @override
  void onMessage(AuthState message) {

>     message.maybeWhen<void>(

Reviewer 2: `.maybeWhen` is specifically generated by `freezed` – consider using a `switch` or an `if` instead
